A01/COMP348-A1-AlexJeanFonz/src/main/java/com/alexjeanfonz/Employee.java

```diff
@@ -89,4 +89,17 @@ public String toString() {
         return String.format("Id: %s, Name: %s %s, Salary: %s", this.id, this.firstName, this.lastName, this.salary);
     }
     
+    // Return an enum representing the employee's salary bracket
```

Comment:
Instead of comments i strongly believe in meta data (javadocs)!
Just type /** and press enter, then fill out the comment form:)